pkg/cloud/ibm/assets/deployment.yaml

```diff
@@ -77,7 +73,6 @@ spec:
               source /etc/kubernetes/apiserver-url.env
             fi
             exec /bin/ibm-cloud-controller-manager \
-            --bind-address=$(POD_IP_ADDRESS) \
```

Comment:
Bind address in the CCM logs is now `0.0.0.0`. I would rather we use `127.0.0.1` explicitly so that we do not expose unauthenticated, non-tls endpoints outside of the host network. Kube RBAC Proxy is in place to make sure the metrics are publicly accessible via an authenticated TLS endpoint

Reply:
Okay, I can switch and do some testing to make sure that still works as expected.